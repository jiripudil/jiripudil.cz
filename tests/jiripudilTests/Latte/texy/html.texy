## Title of the document

Some paragraph.

/--html
<div class="foo">
	<img src="example.png" />
</div>
\--
