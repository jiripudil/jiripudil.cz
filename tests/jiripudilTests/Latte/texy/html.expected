
<h2>Title of the document</h2>

<p>Some paragraph.</p>

<pre><code><span class="html-tag">&lt;div</span><span
class="html-tagin"> class=</span><span
class="html-quote">&quot;foo&quot;</span><span
class="html-tag">&gt;</span>
    <span class="html-tag">&lt;img</span><span
class="html-tagin"> src=</span><span
class="html-quote">&quot;example.png&quot;</span><span
class="html-tagin"> </span><span class="html-tag">/&gt;</span>
<span
class="html-tag">&lt;/div&gt;</span></code></pre>
